content/en/docs/ibm/create-cluster.md

```diff
@@ -132,6 +158,124 @@ Wait until the cluster is deployed and configured. It can take a while for the c
+r006-xxxxxxxx-4002-46d2-8a4f-f69e7ba3xxxx   rising-rectified-much-brew          10.240.0.0/18     us-south-1   false         true         2021-03-05T14:58:39+05:30   
+r006-xxxxxxxx-dca9-4321-bb6c-960c4424xxxx   retrial-reversal-pelican-cavalier   10.240.64.0/18    us-south-2   false         true         2021-03-05T14:58:39+05:30   
+r006-xxxxxxxx-7352-4a46-bfb1-fcbac6cbxxxx   subfloor-certainly-herbal-ajar      10.240.128.0/18   us-south-3   false         true         2021-03-05T14:58:39+05:30  
+```
+
+Now create a subnet as follows.
+
+`$ ibmcloud is subnet-create my-subnet $VPC_ID us-south-3 --ipv4-cidr-block "10.240.128.0/18"`
+
+```shell
+Creating subnet my-subnet in resource group kubeflow under account IBM as user new@user-email.com...
+                       
+ID                  0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   
+Name                my-subnet
+```
+
+List all the existing subnets:
+
+```$ ibmcloud is subnets
+Listing subnets for generation 2 compute in all resource groups and region us-south under account IBM as user prashsh1@in.ibm.com...
+ID                                          Name                      Status      Subnet CIDR       Addresses     ACL                                                    Public Gateway                             VPC                 Zone         Resource group   
+0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   my-subnet                 available   10.240.128.0/18   16373/16384   husker-sloping-bee-resize                              my-gateway                                 my-vpc              us-south-3   kubeflow   
+```
+
+Record the subnet id as,
+`$ export SUBNET_ID=0737-27299d09-1d95-4a9d-a491-a6949axxxxxx`
+
+* Step 5. Create a VPC-gen2 based kubernetes cluster.
+```shell
+$ ibmcloud ks cluster create ${WORKER_NODE_PROVIDER}   --name ${CLUSTER_NAME}   --zone=${CLUSTER_ZONE}   --version=${KUBERNETES_VERSION}   --flavor ${WORKER_NODE_FLAVOR} --vpc-id $VPC_ID --subnet-id $SUBNET_ID --workers=2
+Creating cluster...
+OK
+Cluster created with ID cxxxxxxd00kq9mnxxxxx
+```
+
+* Step 6. Attach a public gateway
+
+This step is mandatory for kubeflow deployment to succeed, because it needs public internet access to download images.
+
+First check, if your cluster is already assigned a public gateway:
+
+```shell
+ibmcloud is pubgws
+Listing public gateways for generation 2 compute in all resource groups and region ...
+ID                                          Name                                       Status      Floating IP      VPC                 Zone         Resource group   
+r006-xxxxxxxx-5731-4ffe-bc51-1d9e5fxxxxxx   my-gateway                                 available   xxx.xxx.xxx.xxx       my-vpc              us-south-3   default   
+
+```
+In the above run, gateway is already attached for the vpc: `my-vpc`. Incase it does not exist, proceed with rest of the setup.
```

Comment:
```suggestion

In the above run, the gateway is already attached for the VPC: `my-vpc`. In case it does not exist, proceed with rest of the setup.
```